text/src/autogluon/text/automm/predictor.py

```diff
@@ -1032,6 +1037,13 @@ def _logits_to_prob(logits: torch.Tensor):
         prob = prob.detach().cpu().float().numpy()
         return prob
 
+    @staticmethod
```

Comment:
Instead of using a static method, let's make it a function and put it in utils or other places.